### DISCUSSION TITLE
Rollup failed to resolve import "highcharts/modules/.js" when running "nuxt build"

### DISCUSSION THREAD

**Discussion by Laureatus** [richardeschloss/nuxt-highcharts#75] [Q&A]
### Installed Versions:
Nuxt 3.10.3
nuxt-highcharts: 3.1.1

### Problem
I have installed nuxt-highcharts and added the dependency in the nuxt.config.ts file like this:
`
export default defineNuxtConfig({
modules: [
    'nuxt-highcharts'
  ],
})`

When I now run `nuxt build` I get the following error: 
` ERROR  Nuxt Build Error: [vite]: Rollup failed to resolve import "highcharts/modules/.js" from "virtual:nuxt:/frontend/.nuxt/nuxt-highcharts.hcMods.js".`

The weird thing is, that there isn't any `nuxt-highcharts.hcMods.js` file inside the `.nuxt `folder. Has anyone an idea what the problem could be and can help me with it?

**Comment by richardeschloss** [comment]
I'm perplexed by this: "highcharts/modules/.js" (empty file name?)

**Reply by JohnMica** [reply]
Me too. 

Not really sure what is happening of it's a rollup thing from Nuxt TBH

**Reply by richardeschloss** [reply]
Fwiw, here's the line of code that adds the hcMods file: 
https://github.com/richardeschloss/nuxt-highcharts/blob/1f8b250eecd79171086f7f8ce3f6756922dec8f2/lib/module.js#L64

That file was added to make it easier to dynamically import the highcharts modules as needed in the browser.

**Reply by richardeschloss** [reply]
Fwiw, I just built master and was able to start successfully. I'm using nuxt v3.6.0 and nuxt-highcharts v3.1.1. What can I tweak in master to repro the issue?

**Reply by JohnMica** [reply]
so far this is what i got whilst upgrading a nuxt2 to nuxt3 project (BTW it used to work a couple of weeks back)

- add  "nuxt-highcharts": "3.1.1", (dep)
- add "nuxt": "3.10.3",  (dev dep)
- run dev (all good)
- run build (errors out with the above error)

It could be that I am doing something wrong ?

**Comment by JohnMica** [comment]
to be fair it seems that using the 'recommended' way from highcharts also provides a strange error

this is when 
- create the plugin.client file
- importing various modules (in this case the dumbbell)
- build and serve

what is even stranger is that in dev mode your package as well as the highcharts-vue package seem to be working

<img width="1556" alt="image" src="https://github.com/richardeschloss/nuxt-highcharts/assets/7076631/e0b36460-01fd-43ac-931f-79a1cb0143ad">
